sys/net/gnrc/application_layer/dhcpv6/client.c

```diff
@@ -89,6 +89,52 @@ void dhcpv6_client_conf_prefix(unsigned iface, const ipv6_addr_t *pfx,
     }
 }
 
+bool dhcpv6_client_check_ia_na(unsigned iface)
+{
+    gnrc_netif_t *netif = gnrc_netif_get_by_pid(iface);
+
+    return netif->ipv6.aac_mode & GNRC_NETIF_AAC_DHCP;
+}
+
+bool dhcpv6_client_add_addr(unsigned iface, ipv6_addr_t *addr)
+{
+    gnrc_netif_t *netif = gnrc_netif_get_by_pid(iface);
+
+    DEBUG("DHCPv6 client: ADD IP ADDRESS\n");
+
+    return gnrc_netif_ipv6_addr_add(netif, addr, 64, 0) > 0;
+}
+
+bool dhcpv6_client_deprecate_addr(unsigned iface, const ipv6_addr_t *addr)
+{
+    gnrc_netif_t *netif = gnrc_netif_get_by_pid(iface);
+
+    for (int i = 0; i < CONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF; i++) {
+        if (ipv6_addr_equal(&netif->ipv6.addrs[i], addr)) {
+            netif->ipv6.addrs_flags[i] &= ~GNRC_NETIF_IPV6_ADDRS_FLAGS_STATE_MASK;
+            netif->ipv6.addrs_flags[i] |= GNRC_NETIF_IPV6_ADDRS_FLAGS_STATE_DEPRECATED;
+            return true;
+        }
+    }
+
+    return false;
+}
+
+bool dhcpv6_client_remove_addr(unsigned iface, const ipv6_addr_t *addr)
+{
+    gnrc_netif_t *netif = gnrc_netif_get_by_pid(iface);
+
+    for (int i = 0; i < CONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF; i++) {
+        if (ipv6_addr_equal(&netif->ipv6.addrs[i], addr)) {
+            gnrc_netif_ipv6_addr_remove_internal(netif,
+                                                 &netif->ipv6.addrs[i]);
```

Comment:
You need to acquire the `netif` to use this function, however, I would rather elect to use the non-internal function. ;-)

Reply:
Should be fixed by 8215aaa819973e1239f41f21bf0324ac091f3e64 now :)